content/mobile-tor/contents.lr

```diff
@@ -0,0 +1,209 @@
+
+* Tap on a button of 3 vertical dots in URL bar.
+* Scroll down and tap on "Security Settings".
+* You can now select an option from the security level slider.
+
+
+### UPDATING
+Tor Browser must be kept updated at all times. If you continue to use an outdated version of the software, you may be vulnerable to serious security flaws that compromise your privacy and anonymity.
+You can update Tor Browser automatically or manually.
+
+#### Updating Tor Browser for Android automatically
+This method assumes that you have either Google Play or F-droid installed on your phone.
+
+##### Google Play
+
+<img style="max-width:300px" class="col-md-6" src="../../static/images/android-update-google-play.png" alt="Updating Tor Browser for Android on Google Play">
+
+Click on the hamburger menu next to the search bar and navigate to "My apps and games" > "Updates".
+If you find Tor Browser on the list of apps which need updating, select it and click the "update" button.
+##### F-Droid
+
+<img style="max-width:300px" class="col-md-6" src="../../static/images/android-update-f-droid.png" alt="Updating Tor Browser for Android on F-droid">
+
+Click on "Settings", then go to "Manage installed apps". 
+On the next screen, select Tor Browser and finally click on the "update" button.
+
+#### Updating Tor Browser for Android manually
+With Tor Browser closed, remove it from your system by [uninstalling](/mobile-tor/#uninstalling) it.
+Visit F-droid, Google Play or the [Tor Project website](https://www.torproject.org/download/#android) and download a copy of the latest Tor Browser release, then [install](/mobile-tor/#) it as before.
+
+
+### UNINSTALLING
+Tor Browser for Android can be uninstalled directly from F-droid, Google Play or from your phone's app settings.
+
+### KNOWN ISSUES
+
+At the moment, there are some features which are not available in Tor Browser for Android, but are currently available in Tor Browser for desktop.
+
+* You can't configure a proxy.
+* You can't see your Tor circuit. [#25764](https://trac.torproject.org/projects/tor/ticket/25764)
+* Custom obfs4 bridges not working. [#30767](https://trac.torproject.org/projects/tor/ticket/30767)
+* 'Clear private data' option does not clear browsing history. [#27592](https://trac.torproject.org/projects/tor/ticket/27592)
+* Tor Browser for Android does not connect when moved to the SD Card.
+* Tor Browser for Android has only private tabs.
+* You can't take screenshots while using Tor Browser for Android. [#27987](https://trac.torproject.org/projects/tor/ticket/27987)
+* You can't use TBA without Tor.
+* You can't configure exit nodes.
+* You can't save images. [#31013](https://trac.torproject.org/projects/tor/ticket/31013)
+* You can't configure bridges while TBA is running. Bridges can only be configured at the launch screen.
+* NoScript permissions can not be configured.
+* Tor Browser for Android does not permit in-app automatic updates.
+
+
+### Tor for iOS
```

Comment:
tor = (also know as little-t-tor): a network program that connects you to the tor network.

Tor Browser = a web browser that connects to the internet using the tor network.

Saying "Tor for iOS" and talking about Onion Browser is mixing tor (the network program) and Tor Browser. So, replace here to "Tor Browser for iOS".